### PR TITLE
Lightmode

### PR DESCRIPTION
**Pull request by Hanttone**
Implemented a light mode into the game and a few animations for the navigation and page transitions. Thanks for your comments! :)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
   },
   "dependencies": {
     "framer-motion": "^2.9.5",
+    "react-input-switch": "^2.2.2",
```
Comment (lmachens) [inline package.json:23]: Are you sure you need this?

**Review by lmachens — COMMENTED** [review]
good job 🎉

File: client/src/App.js [hunk 2]
```diff
@@ -21,20 +24,26 @@ function App() {
     setFetchInProgress,
   ] = useState(true);
 
+  const [mode, setMode] = useState('dark');
+
+  const dataUrls = [
+    'http://localhost:4000/api/scores',
+    'http://localhost:4000/api/germannouns',
+  ];
```
[the remark this answers is not in the document]
Comment (Hanttone) [reply client/src/App.js]: Thanks for the comments, this is already fixed - tried to do a second fetch and ended up not going that path so this all is fixed, also having the actual paths in Master for production, just using localhost for developing. :)

File: client/src/App.js [hunk 3]
```diff
@@ -54,31 +63,40 @@ function App() {
...
+        <AppWrapper>
+          <GlobalStyle />
+          {fetchInProgress ? (
+            <Loading mode={mode} />
+          ) : (
+            <AnimatePresence>
+              <Switch>
                 <Route exact path="/">
                   <motion.div
+                    key="home"
```
[the remark this answers is not in the document]
Comment (Hanttone) [reply client/src/App.js:80]: Yup, without the key the animation is not working. :)

**Comment by Hanttone** [comment]
Merge completed

**Pull request merged**